### ISSUE TITLE
How to save and load model？

### ISSUE THREAD

**Issue by jourmore** [dholzmueller/pytabkit#19]
How to save and load model ？
Thanks ！！！

**Comment by dholzmueller** [comment]
Hi, you can simply use pickle, as in the following example:
```python
import pickle
from sklearn.datasets import load_iris
from sklearn.model_selection import train_test_split

from pytabkit import RealMLP_TD_Classifier

# Example: initialize and train the classifier
clf = RealMLP_TD_Classifier()
X, y = load_iris(return_X_y=True)
X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)

clf.fit(X_train, y_train)

# Save the model to a file
with open('realmlp_model.pkl', 'wb') as f:
    pickle.dump(clf, f)

# Load the model from the file
with open('realmlp_model.pkl', 'rb') as f:
    loaded_clf = pickle.load(f)

# Use the loaded model
accuracy = loaded_clf.score(X_test, y_test)
print(f"Test accuracy: {accuracy:.2f}")
```

**Comment by jourmore** [comment]
> Hi, you can simply use pickle, as in the following example:
> 
> import pickle
> from sklearn.datasets import load_iris
> from sklearn.model_selection import train_test_split
> 
> from pytabkit import RealMLP_TD_Classifier
> 
> # Example: initialize and train the classifier
> clf = RealMLP_TD_Classifier()
> X, y = load_iris(return_X_y=True)
> X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
> 
> clf.fit(X_train, y_train)
> 
> # Save the model to a file
> with open('realmlp_model.pkl', 'wb') as f:
>     pickle.dump(clf, f)
> 
> # Load the model from the file
> with open('realmlp_model.pkl', 'rb') as f:
>     loaded_clf = pickle.load(f)
> 
> # Use the loaded model
> accuracy = loaded_clf.score(X_test, y_test)
> print(f"Test accuracy: {accuracy:.2f}")

Thank you very much ! I have tried the joblib package, but it is a bit large to save the trained models.
